src/tests/Regressions/coreclr/GitHub_116676/test116676.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+public class Test116676
+{
+    [UnsafeAccessor(UnsafeAccessorKind.Method)]
+    extern static void DoesNotExist([UnsafeAccessorType("DoesNotExist")] object a);
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    static void Work(bool f)
+    {
+        if (f)
+            DoesNotExist(null);
+    }
+
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        for (int i = 0; i < 1_000_000; i++)
+        {
+            Console.WriteLine(i);
```

Comment:
This will produce megabytes of output that will get copied around, etc. Can we do something less visible, like `Thread.Sleep(10)` and change the for-loop to go till like 200? We should only need to hit the loop about `CallCountThreshold` times (30) to trigger the background tiered compilation.

Reply:
I don't think it is needed, the test now repros the problem reliably (with the fix from this PR removed)